From 8ee05aae1f0a85bcd763cde1dfcc8a33be85c35c Mon Sep 17 00:00:00 2001
From: Trevor Vaughan <tvaughan@onyxpoint.com>
Date: Wed, 11 Apr 2018 16:01:13 -0400
Subject: [PATCH 25/25] Add cipher and digest difference messages

Ensure that users know that AES is the cipher and SHA is the digest when
CA capabilities are not supported.

Ref #89
---
 src/scepgen-o.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/scepgen-o.c b/src/scepgen-o.c
index 05fc437..7120ade 100644
--- a/src/scepgen-o.c
+++ b/src/scepgen-o.c
@@ -503,7 +503,7 @@ cm_scepgen_o_cooked(struct cm_store_ca *ca, struct cm_store_entry *entry,
 		}
 		if (cipher == cm_prefs_nocipher) {
 			/* Per the latest Draft RFC */
-			cm_log(1, "Could not determine supported CA capabilities, using AES256.\n");
+			cm_log(1, "Could not determine supported CA capabilities, using cipher AES256.\n");
 			cipher = cm_prefs_aes256;
 		}
 	}
@@ -580,7 +580,7 @@ cm_scepgen_o_cooked(struct cm_store_ca *ca, struct cm_store_entry *entry,
 		if (digest == cm_prefs_nodigest) {
 			/* Per SCEP RFC draft-gutmann-scep-10 - March 1, 2018 */
 			/* https://www.ietf.org/id/draft-gutmann-scep-10.txt  */
-			cm_log(1, "Could not determine supported CA capabilities, using SHA256.\n");
+			cm_log(1, "Could not determine supported CA capabilities, using digest SHA256.\n");
 			digest = cm_prefs_sha256;
 		}
 	}
-- 
1.8.3.1

